manager/integration/tests/test_recurring_job.py

```diff
@@ -1239,100 +1224,199 @@ def test_recurring_job_delete(client, batch_v1_beta_api, volume_name):  # NOQA
     wait_for_cron_job_delete(batch_v1_beta_api, JOB_LABEL+"="+back2)
     wait_for_cron_job_delete(batch_v1_beta_api, JOB_LABEL+"="+back3)
 
-    # Should be able to delete recurring job while existing in volume label
+
+@pytest.mark.recurring_job  # NOQA
+def test_recurring_job_delete_should_remove_volume_label(client, batch_v1_beta_api, volume_name):  # NOQA
+    """
+    Scenario: test delete recurring job should remove volume labels
+
+    Given 1 volume created.
+    And  create `snapshot1` recurring job.
+         create `backup1`   recurring job.
```

Comment:
And apply these 2 recurring jobs to the volume.

Reply:
Addressed.